components/dashboard/Dashboard.js

```diff
@@ -218,24 +218,37 @@ export default function Dashboard(props) {
     completedTasks,
     confidentActivityLogEntries,
     historyLimit,
-    recentActivityLogEntries,
-    activityLogEntriesCount,
+    allActivityLogEntries,
     ...restProps
   } = props;
 
-  const [activeDialog, setActiveDialog] = useState();
-  const doneTaskCount = allTaskDispositionEvents.length;
   const tasks = getTasks(props, TASK_COUNT_LIMIT);
-  const totalActivitiesCount = useRef(activityLogEntriesCount);
-  const totalConfidentActivitiesCount = useRef(confidentActivityLogEntries.length);
+  const doneTaskCount = allTaskDispositionEvents.length;
+  const [activeDialog, setActiveDialog] = useState();
 
-  // We only want to update the Gauge once the number of entries has been updated
-  useEffect(() => {
-    if (totalActivitiesCount.current !== activityLogEntriesCount) {
-      totalActivitiesCount.current = activityLogEntriesCount;
-      totalConfidentActivitiesCount.current = confidentActivityLogEntries.length;
-    }
-  }, [activityLogEntriesCount, confidentActivityLogEntries.length]);
+  const confidenceCounts = confidentActivityLogEntries
```

Comment:
Since you are applying the same logic to confidenceCounts and totalCounts, what about pulling the map(....).reduce(...) into a function that you can use for both?

Reply:
👍